apps/frontend/src/app/PageTeam/index.tsx

```diff
@@ -51,6 +51,9 @@ export default function PageTeam() {
   const invalidKey = !database.teams.keys.includes(teamId)
   if (invalidKey) return <Navigate to="/characters" />
 
+  // An edit is triggered whenever a team gets opened even if no edits are done
+  database.teams.set(teamId, { lastEdit: Date.now() })
```

Comment:
Actually, this would be executed every time the page is rendered, not every time a new team is visited. 
Should wrap this in a useeffect to run every time a new teamId is detected.
```suggestion
  useEffect(()=>{
    database.teams.set(teamId, { lastEdit: Date.now() })
  },[teamId])
```